xbmc/filesystem/ShoutcastFile.cpp

```diff
@@ -149,14 +150,28 @@ bool CShoutcastFile::ExtractTagInfo(const char* buf)
   g_charsetConverter.wToUTF8(wConverted, strBuffer);
 
   CRegExp reTitle(true);
-  reTitle.RegComp("StreamTitle=\'(.*?)\';");
+  reTitle.RegComp("StreamTitle=\'(.+?)\';");
 
   if (reTitle.RegFind(strBuffer.c_str()) != -1)
   {
     std::string newtitle(reTitle.GetMatch(1));
     CSingleLock lock(m_tagSection);
-    result = (m_tag.GetTitle() != newtitle);
-    m_tag.SetTitle(newtitle);
+    result = !StringUtils::EqualsNoCase(newtitle, m_tag.GetTitle());
+    if (result)
+    {
+      if ( std::find_if(newtitle.begin(), newtitle.end(), ::islower) == newtitle.end())
+      {
+        // use title case with all caps text
+        StringUtils::ToLower(newtitle);
+        StringUtils::ToCapitalize(newtitle);
+      }
+      m_tag.SetTitle(newtitle);
+      if (!m_cacheReader)
+      {
+        // send updates for non-cached readers
+	      CApplicationMessenger::GetInstance().PostMsg(TMSG_UPDATE_CURRENT_ITEM, 1, -1, static_cast<void*>(new CFileItem(m_tag)));
+	    }
```

Comment:
wrong indentation for the above two lines.

Reply:
Sorry I had to turn off some default on my editor because of historical issues in pvr-nextpvr.  These were corrected in another PR yesterday.

Reply:
??? No merge if not fixed in this PR.